src/services/cartService.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+const cartBaseUrl = '/api/user/cart';
+const addProductToCart = async (dispatch, data, token) => {
+  const postCartUrl = '/api/user/cart';
+  const Headers = {authorization: token};
```

Comment:
try accessing the token from here only, instead of taking it as a argument,like using localstorage this will work perfectly too, its just a suggestion

Reply:
Yes that sounds better, will surely make the change. Thank you for the review.